go/x509/verify_test.go

```diff
@@ -83,6 +84,30 @@ var verifyTests = []verifyTest{
 		errorCallback: expectExpired,
 	},
 	{
+		leaf:              googleLeaf,
+		intermediates:     []string{thawteIntermediate},
+		roots:             []string{verisignRoot},
+		currentTime:       1,
+		dnsName:           "www.google.com",
+		disableTimeChecks: true,
+
+		expectedChains: [][]string{
+			{"Google", "Thawte", "VeriSign"},
+		},
+	},
+	{
+		leaf:              googleLeaf,
+		intermediates:     []string{thawteIntermediate},
+		roots:             []string{verisignRoot},
+		currentTime:       10000000000,
+		dnsName:           "www.google.com",
+		disableTimeChecks: true,
+
+		expectedChains: [][]string{
+			{"Google", "Thawte", "VeriSign"},
```

Comment:
What I mean is a check along the lines of "expired/not-yet-valid cert with disableTimeChecks: false, fails because of validity period" + "same cert with disableTimeChecks: true" passes.